static/src/js/components/ErrorBanner/ErrorBanner.jsx

```diff
@@ -18,30 +17,31 @@ import Row from 'react-bootstrap/Row'
  */
 export const ErrorBanner = ({
   dataTestId,
-  message,
-  previousURL
+  message
 }) => {
-  const currentURL = window.location.pathname
-  const conceptKeywords = ['/collections/', '/variables/', '/services/', '/tools/']
+  const knownCMRLagErrors = [
```

Comment:
doesn't the error boundary have access to the stack trace?  maybe you can check for a specific kind of exception instead of a string comparison, which seems fragile.

Reply:
Here's the stack trace. You can see there are actually two different errors that populate here. I'm looking around for other methods aside from .message I could call on error to get different kinds of exceptions but am coming up blank. Did you have something in mind?
<img width="1719" alt="Screenshot 2024-09-13 at 12 16 53 PM" src="https://github.com/user-attachments/assets/64672864-9641-49e8-b8a7-4050d37baa4d">